server/src/main/java/org/opensearch/plugins/PluginInfo.java

```diff
@@ -561,6 +575,14 @@ public Map<String, List<String>> getIndexActions() {
         return indexActions;
     }
 
+    /**
+     * Parses plugin-permissions.yml file.
+     */
+    @SuppressWarnings("removal")
```

Comment:
I don't see anything in this method or the method it calls that's deprecated. Why this annotation?  Adding a new method with deprecated methods seems wrong... if anything is actually deprecated.

Reply:
removed